### PR TITLE
Updated recipe for yasm/1.3.0

### PR DESCRIPTION
**Pull request by jpc0**
www.tortall.net is inaccessable, the github version from the yasm public repo however has the same checksum and builds correctly

Specify library name and version:  **yasm/1.3.0**

### CODE REVIEW

**Comment by SpaceIm** [comment]
It is accessible.

File: recipes/yasm/all/conandata.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 sources:
   "1.3.0":
-    - url: "http://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz"
+    - url: "https://github.com/yasm/yasm/releases/download/v1.3.0/yasm-1.3.0.tar.gz"
```
Comment (SpaceIm) [inline recipes/yasm/all/conandata.yml:3]:
```suggestion
    - url: [
      "http://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz",
      "https://github.com/yasm/yasm/releases/download/v1.3.0/yasm-1.3.0.tar.gz",
    ]
```

Comment (Croydon) [reply recipes/yasm/all/conandata.yml:3]: Use `https` -> https://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz

**Comment by jpc0** [comment]
> It is accessible. What you can do however is to add a miror in conandata.yml.
> 
> You can also update the title of this PR to refer to updated recipe.

```
Starting Nmap 7.92 ( https://nmap.org ) at 2022-04-13 18:05 SAST
Nmap scan report for www.tortall.net (69.55.235.23)
Host is up (0.32s latency).
rDNS record for 69.55.235.23: 69-55-235-23.in-addr.arpa.johncompanies.com
Not shown: 999 filtered tcp ports (no-response)
PORT   STATE SERVICE
22/tcp open  ssh
```
I'm only getting port 22 open on the host from my end, will update the title a bit later, browser also times out on both http and https, anybody have contact details for a sysadmin there since may be a regional issue from their end then?

DNS resolves to that from Cisco Umbrella, Cloudflare and Google DNS so I don't think there is an issue with DNS resolution.

Will check about adding a mirror as well.

File: recipes/yasm/all/conandata.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 sources:
   "1.3.0":
-    - url: "http://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz"
+    - url: [
+      "http://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz",
```
Comment (tbeu) [inline recipes/yasm/all/conandata.yml:4]:
```suggestion
      "https://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz",
```

Comment (jpc0) [reply recipes/yasm/all/conandata.yml:4]: I'm going to commit this to the PR, I cannot check that it works though.

**Review by uilianries — APPROVED** [review]

**Comment by uilianries** [comment]
@jpc0 Thank you for your PR. Please, ask for access request on the issue #4. It's a requirement for our CI, otherwise, the bot will not start your PR.

**Comment by jpc0** [comment]
Thanks for the information, wasn't aware.

**Pull request closed without merging**